src/views/Logbook/LogbookList.tsx

```diff
@@ -20,13 +21,119 @@ import { analytics } from "~/utils"
 
 import * as styles from "./LogbookList.module.css"
 
-const LogbookList = () => {
-  const { locale } = useLocalization()
+type ItemProps = {
+  logbook: Logbook
+}
+
+const LogbookListContentItem: React.FC<ItemProps> = ({ logbook }) => {
   const isMediumUp = useResponsive("md-up")
   const iconSize = isMediumUp ? "xl" : "xlM"
+
+  return (
+    <div className={styles.listitem}>
+      <div
+        className={styles.avatar}
+        style={{
+          backgroundImage: `url("${logbook.tokenImageURL}")`,
+        }}
+      />
+      <div className={styles.avatarDescription}>
+        <h3>Traveloggers #{logbook.tokenId}</h3>
+        <p>Collectors 9 / Edited {logbook.logs.length}</p>
+      </div>
+      <div className={styles.avatarDetailLink}>
+        <Link to={`/logbooks/${logbook.tokenId}`} language={undefined}>
+          <IconChevonLeft
+            size={iconSize}
+            color="white"
+            className={styles.rotate180}
+          />
+        </Link>
+      </div>
+    </div>
+  )
+}
+
+const LogbookListContent = () => {
   const searchInputRef = useRef<HTMLInputElement>(null)
   const [searchTerm, setSearchTerm] = useState("")
 
+  const { account } = useAccount()
+  const { logbooks, getOwnNFTs, ownNFTs } = useContext(LogbookContext)
+
+  useEffect(() => {
+    getOwnNFTs()
```

Comment:
```
if (!account) {
  return
}
...
```
?

Reply:
it seems safe because `getOwnNFTs` has that check builtin:

https://github.com/thematters/traveloggers-app/blob/d0379785b07b545768f32c6f3b0c3036339d2333/src/components/LogbookContext/index.tsx#L249-L254